internal/pkg/manifest/validate.go

```diff
@@ -114,18 +120,24 @@ func (b *BackendServiceConfig) Validate() error {
 			return fmt.Errorf(`validate "taskdef_overrides[%d]": %w`, ind, err)
 		}
 	}
+	name := aws.StringValue(b.Name)
+	if name == "" {
+		return &errFieldMustBeSpecified{
+			missingField: "name",
+		}
+	}
 	if err = validateContainerDeps(validateDependenciesOpts{
 		sidecarConfig:     b.Sidecars,
 		imageConfig:       b.ImageConfig.Image,
-		mainContainerName: b.name,
+		mainContainerName: name,
 	}); err != nil {
 		return fmt.Errorf("validate container dependencies: %w", err)
 	}
 	return nil
 }
 
 // Validate returns nil if RequestDrivenWebService is configured correctly.
-func (r *RequestDrivenWebServiceConfig) Validate() error {
+func (r *RequestDrivenWebService) Validate() error {
```

Comment:
Should we validate `name` for RDWS as well?
```suggestion
func (r *RequestDrivenWebService) Validate() error {
      name := aws.StringValue(r.Name)
	if name == "" {
		return &errFieldMustBeSpecified{
			missingField: "name",
		}
	}
```